maintenance-calendar/modules/ecs_scans/lambda/src/ignore.ts

```diff
@@ -0,0 +1,148 @@
+// There are some vulnerabilities that, from the descriptions, clearly do not
+// apply to us. If we don't do something about these, we would trigger alerts
+// every day for every cluster we scan. In these cases, we can add the
+// vulnerability info to parameter store so that it will be ignored by the scan.
+//
+// We can add the cluster name and a timestamp to another parameter to temporarily
+// snooze _all_ alerts on that cluster. Valid for timestamp + seven days.
+// Intended for cases where a vulnerability is patched but deployment is delayed
+
+import { Attribute, ImageScanFinding } from "@aws-sdk/client-ecr";
+import { SSMClient, GetParameterCommand } from "@aws-sdk/client-ssm";
+import assert from "assert";
+import pino from "pino";
+
+const logger = pino({ level: process.env.LOG_LEVEL ?? "debug" });
+const ssmClient = new SSMClient();
+
+type IgnoreSpec = {
+  name: string;
+  packageName: string;
+  packageVersion?: Array<string>;
+};
+
+type SnoozeList = {
+  cluster: string;
+  date: string;
```

Comment:
How would you feel about the structure of this being like this?
```ts
{
  cluster: string;
  snoozeUntil: string;
}
```
My thinking here is that 1) `date` is rather ambiguous as a property name and 2) the snooze ending at `snoozeList.date + 7 days` is a hidden behavior if you haven't read this code whereas `snoozeList.snoozeUntil` is immediately intelligible if you're poking around in parameter store

Reply:
Also this is nitpicky as fuck but shouldn't it be called `SnoozeListItem` since the param is actually a list of these doohickeys?